CHANGELOG.md

```diff
@@ -16,23 +16,25 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ## Unreleased
 
 ## Added
-
+- Ability to use Social Media molecule as a Wagtail module in the Sidefoot
```

Comment:
Lots of unrelated changes to the log, are they all making up for existing changes that weren't documented?